### DISCUSSION TITLE
[New Feature] Transport door open/close animation

### DISCUSSION THREAD

**Discussion by Bittah** [Vinifera-Developers/Vinifera#1425] [Feature Requests]
### Description:
Transport units in TD and RA1 have an animation that shows their door open/close before passengers enter or exit it, but this is missing from TS.

### Possible Implementation:
Add these keys for SHP units in Art.ini:
 ```ini
DoorOpeningFrames=
DoorOpenFrames=
DoorClosingFrames=
StartDoorOpeningFrame=
StartDoorOpenFrame=
StartDoorClosingFrame=
```
`DoorOpenFrames=` will basically function the same as `StandingFrames=` when the door is open, while the other keys control the transitions between `DoorOpenFrames` and the `StandingFrames`.

**Comment by Rampastring** [comment]
Gameplay-wise I think TS is better here, as in TD and RA the units wait for the door to open before entering, which makes getting on board a rather clumsy and slow process. It could be nice if this was a purely visual thing (or configurable to be such) and units were allowed to board the transport regardless of the door state.
